ios/ReactNativeCameraKit/CameraView.swift

```diff
@@ -184,14 +186,16 @@ class CameraView: UIView {
 
         // Scanner
         if changedProps.contains("scanBarcode") || changedProps.contains("onReadCode") {
+            let supportedFormats = supportedBarcodeType.map { CodeFormat.fromAVMetadataObjectType($0) }
```

Comment:
We can use `CodeFormat` for `supportedBarcodeType` line 23, so no need to map it here

Reply:
Oh yes I should have just done that :D

Reply:
@DavidBertet  check now